configure.ac

```diff
@@ -497,6 +497,9 @@ AS_IF([test "x$with_julia" != xno ],[
   AS_IF([ test $? != 0 ], [AC_MSG_ERROR([failed to obtain JULIA_LIBS from julia-config.jl])])
   JULIA_LIBS=${JULIA_LIBS//\'/}
   AC_MSG_RESULT([${JULIA_LIBS}])
+  AC_MSG_NOTICE([installing GAPTypes.jl])
+  ${JULIA} -e 'import Pkg; Pkg.add(Pkg.PackageSpec(url="https://github.com/oscar-system/GAPTypes.jl", rev="master"))'
```

Comment:
Is this meant to be the permanent form of the patch? Or is it meant to just do `Pkg.add("GAPType.jl")` eventually? I guess there are some merits to either... But I am not exactly comfortable with referencing `master` here, at least on the long run. Of course for now, while we develop stuff, we can do that; but ultimately, I'd expect this to reference a specific tag? Although now that I think about it, maybe not, so that we can hotfix things? Maybe a branch???

Reply:
We should probably explain whatever the plan is in a brief comment here.

Reply:
This is meant to be replaced once the release of GAPTypes.jl is accepted.